tests/ttnn/unit_tests/operations/eltwise/test_unary.py

```diff
@@ -1887,3 +1887,64 @@ def test_unary_root_ops_ttnn(input_shapes, torch_dtype, ttnn_dtype, ttnn_op, fas
                 torch.isinf(output_tensor), torch.tensor(float("nan"), dtype=output_tensor.dtype), output_tensor
             )
         torch.isclose(output_tensor, golden_tensor, equal_nan=True)
+
+
+@pytest.mark.parametrize(
+    "param",
+    {-1.5, 1.7, 0.0},
+)
+@pytest.mark.parametrize("round_mode", [None, "trunc", "floor"])
```

Comment:
```suggestion
@pytest.mark.parametrize("rounding_mode", [None, "trunc", "floor"])
```
Please address these throughout the PR. Thanks!